Packs/SpyCloudPack/Integrations/SpyCloud/README.md

```diff
@@ -0,0 +1,9 @@
+# SpyCloud Integration
+Integration for the SpyCloud ATO service. Retrieve breach data for
+- Domain searches
+- Email searches
+- Breaches
+
+Filter arguments are added as well.
+
+**Make sure to whitelist your outbound IP address XSOAR uses to connect**
```

Comment:
Please modify the **Integration** readme to meet our standards:
https://xsoar.pan.dev/docs/documentation/readme_file